federation-js/src/composition/validate/postComposition/__tests__/externalTypeMismatch.test.ts

```diff
@@ -36,10 +38,22 @@ describe('validateExternalDirectivesOnSchema', () => {
       Array [
         Object {
           "code": "EXTERNAL_TYPE_MISMATCH",
+          "locations": Array [
```

Comment:
Can this location point to the return type of the field instead?

Reply:
Since the return type doesn't have an ast node, can we do that? externalField.type would point to the node of the definition of the type.